E2E/assertions/assertions.ts

```diff
@@ -0,0 +1,40 @@
+const { I, game } = inject();
+
+export = {
+    async checkIfTheLinkWasCopied (currentUrl) {
+      I.usePlaywrightTo('check clipboard', async ({ page, browserContext }) => {
+        await browserContext.grantPermissions(['clipboard-read']);
+        const clipboardText = await page.evaluate("navigator.clipboard.readText()");
+        I.assertEqual(currentUrl, clipboardText);
+      });
+  },
+
+  async checkIfTheSPIsRoundedCorrectlyForTheFirstPlayer() {
+    const averageFirstPlayerSP = game.calculateAverage("bishop", "d", 3);
+    let expectedFirstPlayerScore = game.checkPlayerSP(averageFirstPlayerSP);
+    let actualFirstPlayerSP = Number(await I.grabTextFrom(game.locator.playersList.playerIndividualSP(2)));
+    I.assertEqual(actualFirstPlayerSP, expectedFirstPlayerScore );
+  },
+
+  async checkIfTheSPIsRoundedCorrectlyForTheSecondPlayer() {
+    const averageSecondPlayerSP = game.calculateAverage("queen", "f", 5);
+    let expectedSecondPlayerScore = game.checkPlayerSP(averageSecondPlayerSP);
+    let actualSecondPlayerSP = Number(await I.grabTextFrom(game.locator.playersList.playerIndividualSP(1)));
+    I.assertEqual(actualSecondPlayerSP, expectedSecondPlayerScore);
+  },
+
+  async checkIfTheFinalSPIsRoundedCorrectly() {
```

Comment:
Does it only allows to assert result if there are 2 players?

Reply:
Yes